src/iconsManifest/languages.ts

```diff
@@ -31,6 +31,7 @@ export const languages: ILanguageCollection = {
   c_al: { ids: 'c-al', defaultExtension: 'cal' },
   cabal: { ids: 'cabal', defaultExtension: 'cabal' },
   caddyfile: { ids: 'caddyfile', defaultExtension: 'Caddyfile' },
+  ceylon: { ids: 'ceylon', defaultExtension: 'ceylon' },
```

Comment:
Can you point us out to the extension supporting this language id?

Reply:
Already mentioned in the issues OP https://github.com/vscode-icons/vscode-icons/issues/2095

Reply:
Totally missed that!